README.md

```diff
@@ -1,24 +1,24 @@
 docker-rotate
 =============
 
-In a continuously deployed environment, old and unused docker images accumulate and use up space.
-`docker-rotate` helps remove the K oldest images of each type.
+In a continuously deployed environment, old and unused docker images and containers accumulate and use up space.
```

Comment:
That's what docker-rotate is doing - clearing out unused images and non-running containers to clear up space on the host